### PR TITLE
Portland 2024 CFP

### PR DESCRIPTION
**Pull request by plaindocs**
I don't think the text of what we're looking for needs much or any change.

- [x] blog post
- [x] Pretalx(I got midway to doing this and ran into a bug I'm hoping they will fix)
- [x] Oh, are tickets on sale yet? Or 2024 only? 

-----

OK, I'd love eyes on the blog post, and on the form itself, but otherwise it's good to go with a publish date of Monday.

----
:books: Documentation preview :books:: https://writethedocs-www--2045.org.readthedocs.build/

### CODE REVIEW

**Comment by ericholscher** [comment]
@plaindocs was thinking tickets in December, so folks can expense them with 2023 funds, was the plan.

File: docs/conf/portland/2024/news/announcing-cfp.rst [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+----------------------------------------
+
+If you want to speak at Write the Docs, but can't or don't want to attend an in-person event, keep an eye out for Write the Docs Atlantic, an online conference coming later in 2024.
+
+This conference will be entirely virtual, in a timezone that works for as many people as possible, so you can present or attend from anywhere.
+
+Tickets
+-------
+
+Tickets will officially go on sale early **2024**.
```
Comment (ericholscher) [inline docs/conf/portland/2024/news/announcing-cfp.rst]:
```suggestion
Tickets will go on sale {{ tickets_live }}.
```

File: docs/conf/portland/2024/news/announcing-cfp.rst [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+As always, we keep ticket prices low so that the event is accessible to the widest range of people possible.
+However, if you can't afford the tickets and still wish to attend, we will launch our grants program soon after opening ticket sales.
+
+Stay Updated
```
Comment (ericholscher) [inline docs/conf/portland/2024/news/announcing-cfp.rst:61]: We should use this announcement to ship our sponsorship program, which is likely the other thing we should be highlighting. Will work on that this week.

**Review by ericholscher — COMMENTED** [review]
Looks good to me 💯

File: docs/conf/portland/2024/news/announcing-cfp.rst (hunk 2 above)
Comment (plaindocs) [reply docs/conf/portland/2024/news/announcing-cfp.rst:61]: Yep, lemme know if you need a hand. Will just need to update the blog post date when we do put it out.

**Comment by ericholscher** [comment]
Alright, pushed up a small update with the sponsorship info 👍

File: docs/conf/portland/2024/news/announcing-cfp.rst [hunk 3]
```diff
@@ -0,0 +1,70 @@
+:template: {{year}}/generic.html
+
+.. post:: Nov 06, 2023
```
Comment (ericholscher) [inline docs/conf/portland/2024/news/announcing-cfp.rst]:
```suggestion
.. post:: Nov 14, 2023
```

**Comment by ericholscher** [comment]
Alright.. think this is ready to go!

**Pull request merged**